### DISCUSSION TITLE
SyntaxError: Invalid or unexpected token @import

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#3414] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19658624/160039836-12c43142-86a0-4398-b3df-f8751f9e13e9.png)

**Comment by aruniverse** [comment, marked as answer]
How are you building your frontend and running that file?

**Reply by Snailya** [reply]
build with tsc and run with npm run command
![image](https://user-images.githubusercontent.com/19658624/160041415-12c1f2a2-cc69-471f-849f-f97977a146e6.png)
tsconfig is as below:
![image](https://user-images.githubusercontent.com/19658624/160041466-510892a8-34ae-4277-ab8d-14a86ea2163b.png)

**Reply by aruniverse** [reply]
Typescript will only compile your ts code. It won't touch any of the scss files. You need to use a tool like webpack to bundle your frontend together. Any reason you aren't using @bentley/react-scripts?

**Reply by Snailya** [reply]
Thanks. Indeed, I don't know that I need to create project using @bentley/react-scripts.
The tutorial is unfriendly for a starter. Also the framework of itwin is too complex.

**Reply by ColinKerr** [reply]
@Snailya, which tutorial did you use?
